dataquality/loggers/data_logger/text_ner.py

```diff
@@ -401,14 +400,16 @@ def _log_dict(
         )
 
     def _log_df(
-        self, df: Union[pd.DataFrame, DataFrame], meta: List[Union[str, int]]
+        self,
+        df: Union[pd.DataFrame, DataFrame],
+        meta: Union[List[str], List[int], None],
```

Comment:
Should default to `None`, right?
```suggestion
        meta: Union[List[str], List[int], None] = None,
```

Reply:
doesn't have to

Reply:
Why not?